### PR TITLE
Fixes #750 : fixes crash when delete + scroll

### LINKED ISSUE #750: library/bookshelf: delete publication, scroll (press arrow) => fatal crash (white window)
```
Slider.tsx:57 Uncaught TypeError: Cannot read property 'getElementsByTagName' of null
    at contentElRefs.map (Slider.tsx:57)
    at Array.map (<anonymous>)
    at Slider.componentDidUpdate (Slider.tsx:56)
    at commitLifeCycles (react-dom.development.js:21983)
    at commitLayoutEffects (react-dom.development.js:25291)
    at HTMLUnknownElement.callCallback (react-dom.development.js:362)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:411)
    at invokeGuardedCallback (react-dom.development.js:466)
    at commitRootImpl (react-dom.development.js:25027)
    at unstable_runWithPriority (scheduler.development.js:815)
contentElRefs.map @ Slider.tsx:57
componentDidUpdate @ Slider.tsx:56
commitLifeCycles @ react-dom.development.js:21983
commitLayoutEffects @ react-dom.development.js:25291
callCallback @ react-dom.development.js:362
invokeGuardedCallbackDev @ react-dom.development.js:411
invokeGuardedCallback @ react-dom.development.js:466
commitRootImpl @ react-dom.development.js:25027
unstable_runWithPriority @ scheduler.development.js:815
runWithPriority$2 @ react-dom.development.js:12188
commitRoot @ react-dom.development.js:24868
finishSyncRender @ react-dom.development.js:24254
performSyncWorkOnRoot @ react-dom.development.js:24226
(anonymous) @ react-dom.development.js:12238
unstable_runWithPriority @ scheduler.development.js:815
runWithPriority$2 @ react-dom.development.js:12188
flushSyncCallbackQueueImpl @ react-dom.development.js:12233
flushSyncCallbackQueue @ react-dom.development.js:12221
discreteUpdates$1 @ react-dom.development.js:24368
discreteUpdates @ react-dom.development.js:1486
dispatchDiscreteEvent @ react-dom.development.js:5926
console-redirect.js:73
```
```
The above error occurred in the <Slider> component:
    in Slider (created by withTranslator(Slider))
    in withTranslator(Slider) (created by GridView)
    in section (created by GridView)
    in GridView (created by Catalog)
    in main (created by LibraryLayout)
    in div (created by LibraryLayout)
    in LibraryLayout (created by Context.Consumer)
    in withRouter(LibraryLayout) (created by ConnectFunction)
    in ConnectFunction (created by Catalog)
    in Catalog (created by Context.Consumer)
    in withRouter(Catalog) (created by withTranslator(withRouter(Catalog)))
    in withTranslator(withRouter(Catalog)) (created by Context.Consumer)
    in Route (created by PageManager)
    in Switch (created by PageManager)
    in PageManager (created by Dropzone)
    in div (created by Dropzone)
    in Dropzone (created by App)
    in div (created by App)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Context.Consumer)
    in ConnectedRouterWithContext (created by ConnectFunction)
    in ConnectFunction (created by App)
    in Provider (created by App)
    in App
```
```
Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
newConsoleFunction @ console-redirect.js:73
logCapturedError @ react-dom.development.js:21707
logError @ react-dom.development.js:21744
update.callback @ react-dom.development.js:23112
callCallback @ react-dom.development.js:13862
commitUpdateEffects @ react-dom.development.js:13900
commitUpdateQueue @ react-dom.development.js:13888
commitLifeCycles @ react-dom.development.js:22026
commitLayoutEffects @ react-dom.development.js:25291
callCallback @ react-dom.development.js:362
invokeGuardedCallbackDev @ react-dom.development.js:411
invokeGuardedCallback @ react-dom.development.js:466
commitRootImpl @ react-dom.development.js:25027
unstable_runWithPriority @ scheduler.development.js:815
runWithPriority$2 @ react-dom.development.js:12188
commitRoot @ react-dom.development.js:24868
finishSyncRender @ react-dom.development.js:24254
performSyncWorkOnRoot @ react-dom.development.js:24226
(anonymous) @ react-dom.development.js:12238
unstable_runWithPriority @ scheduler.development.js:815
runWithPriority$2 @ react-dom.development.js:12188
flushSyncCallbackQueueImpl @ react-dom.development.js:12233
flushSyncCallbackQueue @ react-dom.development.js:12221
discreteUpdates$1 @ react-dom.development.js:24368
discreteUpdates @ react-dom.development.js:1486
dispatchDiscreteEvent @ react-dom.development.js:5926
Slider.tsx:57 Uncaught TypeError: Cannot read property 'getElementsByTagName' of null
    at contentElRefs.map (Slider.tsx:57)
    at Array.map (<anonymous>)
    at Slider.componentDidUpdate (Slider.tsx:56)
    at commitLifeCycles (react-dom.development.js:21983)
    at commitLayoutEffects (react-dom.development.js:25291)
    at HTMLUnknownElement.callCallback (react-dom.development.js:362)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:411)
    at invokeGuardedCallback (react-dom.development.js:466)
    at commitRootImpl (react-dom.development.js:25027)
    at unstable_runWithPriority (scheduler.development.js:815)
```

### PR DESCRIPTION
**Pull request by Scarsniik**
fixes #750

### CODE REVIEW

File: src/renderer/components/utils/Slider.tsx [hunk 1]
```diff
@@ -54,12 +54,14 @@ class Slider extends React.Component<Props, State> {
     public componentDidUpdate() {
         if (this.state.refreshVisible) {
             this.contentElRefs.map((element, index) => {
-                const buttonList = element.getElementsByTagName("button");
-                for (const button of buttonList) {
-                    if (!this.isElementVisible(index)) {
-                        button.tabIndex = "-1";
-                    } else {
-                        button.tabIndex = "0";
+                if (element) {
```
Comment (danielweck) [inline src/renderer/components/utils/Slider.tsx]: To help future developers / codebase reviewers, I think we need to add a code comment explaining why the `this.contentElRefs` array can contain `null` items.

My suggestion:

> The `this.contentElRefs` array is automatically populated in the `render()` > `createContent()` function, via the `div` element's `ref` callback (`ref={(ref) => this.contentElRefs[index] = ref}`), which can be invoked with `null` during the element's "unmount" lifecycle (see https://reactjs.org/docs/refs-and-the-dom.html ). Consequently, we need to check for possibly-`null` values in the `this.contentElRefs` array, in this `componentDidUpdate()` function. However, we can safely ignore usages of `this.contentElRefs` in the `moveInView()` and `isElementVisible()` functions, as these are guaranteed to be invoked when the element is still "mounted" (see the `onFocus` callback).

What do you think? Does that make sense?

Comment (Scarsniik) [reply src/renderer/components/utils/Slider.tsx]: Yese, it seems to be good. I add it

Comment (panaC) [reply src/renderer/components/utils/Slider.tsx]: I propose also to type contentElRefs, it can be helpful

Comment (danielweck) [reply src/renderer/components/utils/Slider.tsx]: Yes, typing would be good, BUT in this case: let's merge this critical bugfix, as we are now getting closer to a release milestone.

**Pull request merged**